src/main/java/core/TrelloServiceObject.java

```diff
@@ -0,0 +1,137 @@
+package core;
+
+import static org.hamcrest.Matchers.lessThan;
+
+import beans.TrelloBoardResponse;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.http.HttpStatus;
+
+public class TrelloServiceObject extends TrelloProperties {
```

Comment:
Use "extends" if the relation between classes is "is a" - Cat is an animal - Cat extends Animal
TrelloServiceObject is NOT  Trello properties
Use static imports when you need constants

Reply:
Fixed.